src/Compilers/CSharp/Portable/Parser/SlidingTextWindow.cs

```diff
@@ -374,6 +374,26 @@ public char PeekChar(int delta)
             return ch;
         }
 
+        public char PreviousChar()
+        {
+            Debug.Assert(this.Position > 0);
+            var desiredPosition = this.Position - 1;
+            if (desiredPosition >= _basis && _offset > 1)
```

Comment:
>_offset > 1 [](http://example.com/codeflow?start=45&length=11)

Can this ever be false?

Reply:
yes.  it's def possible for `_offset` to be zero.  In the case we're testing, that's what is actually happening.